internal.nix

```diff
@@ -331,6 +331,9 @@ rec {
 
               ${preInstallLinkCommands}
 
+              jq -r 'select(.bin != null) | .bin | if type == "string" then . else values[] end' package.json | while read binTarget; do
+                chmod +x "$binTarget" # patchShebangs will only patch executable files
```

Comment:
maybe this should throw if binTarget is not found

Reply:
> npmlock2nix should ignore bin's for the root package (/tmp/package)

even more, npmlock2nix should ignore install scripts for the root package,
since for the root package, only the *.json files are available,
and install scripts may require other files

Reply:
Is that so? I am not entirely sure that is true. Right now we do an `npm install` in the `node_modules` build and that does what we need (most of the time?).

Reply:
consider this package.json

```json
{
  "name": "root-package",
  "version": "1",
  "scripts": {
    "postinstall": "node some/script.js"
  },
  "dependencies": {
    "foo": "1.2.3"
  }
}
```

npmlock2nix has only two files for the root-package: package.json and package-lock.json

these are symlinked to /build in npmlock2nix.node_modules

https://github.com/nix-community/npmlock2nix/blob/33eb3300561d724da64a46ab8e9a05a9cfa9264b/internal.nix#L365-L368

so `some/script.js` is NOT available in /build, and `npm install` would fail